README.md

```diff
@@ -0,0 +1,27 @@
+# Sern CLI
+
+Our CLI is allowing you to create & streamline your Discord bot project without writing a single line of code!
+
+😁 **User Friendly Usage** <br>
+🔥 **At blazing Speeds** <br>
+🌱 **Efficiently** <br>
+
+## Installation
+
+```sh
+npm install @sern/cli@latest
+```
+
+```sh
+yarn add @sern/cli@latest
+```
+
+## Getting Started
+
+When you install the CLI, you can use our commands with **sern** prefix. Usage is simple;
```

Comment:
end of line : use a colon ( : ) instead of semicolon ( ;)

Reply:
Removed it